AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -192,11 +192,25 @@ public class CardComponent: PresentableComponent,
         formViewController.delegate = self
         formViewController.cardDelegate = self
         formViewController.title = paymentMethod.displayInformation(using: configuration.localizationParameters).title
+        
+        formViewController.items.onDidTriggerEvent = { [weak self] infoType, target in
+            self?.sendInfoEvent(of: infoType, target: target)
+        }
+        
         return formViewController
     }()
     
     private let panThrottler = Throttler(minimumDelay: CardComponent.Constant.secondsThrottlingDelay)
     private let binThrottler = Throttler(minimumDelay: CardComponent.Constant.secondsThrottlingDelay)
+    
+    private func sendInfoEvent(of type: AnalyticsEventInfo.InfoType, target: AnalyticsEventTarget) {
+        var infoEvent = AnalyticsEventInfo(
```

Comment:
I would suggest at least extracting the instantiation of events to a separate entity. It could be just an extension for AnalyticsEvent with a set of static functions. It would make any refactoring easier in the future when we for example would swift from three distinct AnalyticEvent types to a single one (assuming this happens sometime in the future). No action needed, just a suggestion for the future improvements.